@xen-orchestra/backups/_runners/_vmRunners/_AbstractXapi.mjs

```diff
@@ -257,6 +265,28 @@ export const AbstractXapi = class AbstractXapiVmBackupRunner extends Abstract {
         }
       })
     })
+
+    // now that we use CBT, we can destroy the data of the snapshot used for this backup
+    // going back to a previous version of XO not supporting CBT will create a full backup
+    // this will only do something after snapshot and transfer
+    if (
+      this._exportedVm?.is_a_snapshot &&
+      this._settings.snapshotRetention === 0 &&
+      this._settings.preferNbd &&
```

Comment:
I think we should add a comment to explain that in some cases we could have `this._settings.preferNbd = true` but NBD not used (even though it seems too corner-case to need a fix)